include/zephyr/bluetooth/classic/a2dp.h

```diff
@@ -30,34 +30,33 @@ extern "C" {
 /** MPEG2,4 IE length */
 #define BT_A2DP_MPEG_2_4_IE_LENGTH (6u)
 /** The max IE (Codec Info Element) length */
-#define A2DP_MAX_IE_LENGTH (8U)
+#define A2DP_MAX_IE_LENGTH         (8U)
```

Comment:
There's still inconsistency here. Just look at the defines above this. They all use lower-case `u`. Furthermore, there's also a namespace violation: this is a public Bluetooth header file, so all defines and symbols must use a `BT_*` or `bt_*` prefix.

Reply:
fixed.

Reply:
> fixed.

No, there's still `A2DP_MAX_IE_LENGTH` which doesn't have a `BT_` prefix.